client/src/modules/Score/components/ScoreTable/index.tsx

```diff
@@ -38,6 +38,22 @@ type StudentsState = {
 
 const courseTasksApi = new CoursesTasksApi();
 
+function trimFiltersProps(filters: ScoreTableFiltersModified): ScoreTableFiltersModified {
+  const trimmedFiltersProps = structuredClone(filters);
+  const filtersKeys = Object.keys(filters) as (keyof ScoreTableFiltersModified)[];
+
+  filtersKeys.forEach(key => {
+    const filterKey = filters[key];
+    if (filterKey && typeof filterKey !== 'boolean') {
+      key = key as Exclude<keyof ScoreTableFiltersModified, 'activeOnly'>;
+      if (Array.isArray(filterKey)) {
+        trimmedFiltersProps[key] = filterKey.map(elem => elem.trim());
+      }
+    }
+  });
+  return trimmedFiltersProps;
+}
```

Comment:
Please consider to do it somehow like this
```suggestion
function trimFiltersProps(
  filters: ScoreTableFiltersModified
): ScoreTableFiltersModified {
  const trimmedFiltersProps = structuredClone(filters);

  Object.entries(filters).forEach(([filterKey, filterValue]) => {
    if (filterValue && typeof filterValue !== 'boolean') {
      if (Array.isArray(filterValue)) {
        trimmedFiltersProps[
          filterKey as keyof ScoreTableFiltersModified
        ] = filterValue.map(elem => elem.trim());
      }
    }
  });

  return trimmedFiltersProps;
}
```

Reply:
Wouldn't it be simpler to fix this here and get rid of this bug in many places at once?
https://github.com/rolling-scopes/rsschool-app/blob/master/client/src/utils/queryParams-utils.ts

Reply:
I have already remove this function